projects/plugins/jetpack/modules/woocommerce-analytics/classes/class-jetpack-woocommerce-analytics-checkout-flow.php

```diff
@@ -124,10 +141,26 @@ public function capture_cart_view() {
 	 * Track the checkout page view
 	 */
 	public function capture_checkout_view() {
-		if ( ! is_checkout() ) {
+		global $post;
+		$checkout_page_id = wc_get_page_id( 'checkout' );
+
+		$is_checkout = $checkout_page_id && is_page( $checkout_page_id )
```

Comment:
Do we also need to account for the classic shortcode block here too?

Reply:
Good catch, fixed.